quote.py

```diff
@@ -66,21 +66,21 @@ def quoteadd(self, quote):
 				with open(self.path, 'a') as f:
 					print(quote, file=f)
 			except (IOError, OSError) as e:
-				return 'Couldn\'t add quote! I\'ll remember it for now, though :) ({})'.format(e)
+				return "Couldn't add quote! I\'ll remember it for now, though :) ({})".format(e)
```

Comment:
There is still an escaped quote here! Blocking!

Reply:
Ugh that's what I get for rushing through this change. I bet it won't even compile.